Dockerfile

```diff
@@ -22,9 +22,11 @@ RUN dnf update -y --setopt='tsflags=nodocs' && \
     dnf install -y binutils
 
 COPY ./ /tmp/package-extract
-RUN cd /tmp/package-extract && \
-    make all && \
-    rm -rf /tmp/package-extract
+RUN pip install micropipenv \
+        && cd /tmp/package-extract \
+        && micropipenv install \
+        && pip install --no-deps . \
```

Comment:
why is this required ?

Reply:
Indeed we should.
You mean the pip part ?
Micropipenv can't currently handle 'build + install' process (`micropipenv install .` result in a never-ending loop) so we just use it to install the exact deps  (specified in Pipfile.lock) and tell pip to build and install the Python package in the current dir.

Reply:
acknowledge thanks.